source/content/guides/composer-convert.md

```diff
@@ -147,13 +156,22 @@ cp -r /themes/custom/great_theme ../$site-composer/web/themes/custom
 Follow suit with any other custom code you need to carry over.
 
 #### Settings.php
+
 Your existing site may have customizations to `settings.php` or any other config files. Review these carefully and extract relevant changes from these files to copy over. Always review any file paths referenced in the code, as these paths may change in the transition to Composer.
 
-It is not wise to completely overwrite the  `settings.php` file with the old one, as there are customizations for moving the configuration directory you don't want to overwrite, as well as platform specific customizations.
+It is not wise to completely overwrite the `settings.php` file with the old one, as there are customizations for moving the configuration directory you don't want to overwrite, as well as platform specific customizations.
+
+```bash{promptUser:user}
+git checkout master sites/default/settings.php
+mv -f sites/default/settings.php web/sites/default/
+git rm -rf sites
+git diff
```

Comment:
The irony here is that the below commands do what the above suggestion says is not wise. Here's an alternative incantation that's more in keeping with "extract relevant changes from these files"
```suggestion
# Ensure working tree is clean
git status
git checkout master
git checkout -b old-settings-php
git mv sites/default/settings.php web/sites/default
git commit -m 'Relocate settings.php'
git checkout composerify
# Interactively extract relevant hunks from old settings.php
git checkout -p old-settings-php web/sites/default/settings.php
```